test/build.t

```diff
@@ -2,14 +2,24 @@
 
 Building the package for aurget
 
-  $ aurget -Sb --builddir "$PWD" --noedit --nodeps --noconfirm --nodiscard aurget
+  $ aurget -Sb --builddir "$PWD" --noedit --nodeps --noconfirm --nodiscard --debug --nosign aurget
```

Comment:
I think I would take the `--debug` back out of here. I can see it being useful to verify _this_ PR as you worked, but now that you're done, I think it's not worth the maintenance burden going forward. Assertions against something like debug logging are a bad idea because they're brittle: changes in them would have nothing to do with functionality but would cause the test to fail anyway.

WDYT?

Reply:
About the debug flag, you are right, that should probably be not in the tests, since it will cause to much trouble. 
It might be something useful though to see the `$makepkg_options` before `makepkg` is called, without having `debug` enabled. But that is something for the future I guess, I will remove the test commit. 

Do you think we should add another test where we only check the debug line of the `makepkg_options` and `(glob)` the rest?

Reply:
> Do you think we should add another test where we only check the debug line of the makepkg_options and (glob) the rest?

Do we have any tests that attempt to pass `makepkg` options along? If we did, then I'd conser this covered already since the fixture system records commands with their options passed -- if that code regressed, we'd get a "no fixture" error. Not the most useful failure, but it is good regression coverage IMO.

Reply:
I think most of the builtin options are tested, but for example custom flags are not tested yet. Maybe it would be good to add one as I did in the test.